src/shared/ui/buttons.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as path from 'path'
+import { ExtensionContext, QuickInputButton, Uri } from 'vscode'
+
+let resourcesAbsolutePath: string | undefined
+
+/**
+ * Loads the extension's absolute path so the help button can find its icon
+ * @param path Extension's absolute path
+ */
+export function initializeButtons(context: ExtensionContext): void {
+    resourcesAbsolutePath = context.asAbsolutePath('resources')
```

Comment:
We should try to avoid global init functions like this where possible. The init wizard is only used in `createNewSamApp`, which already had the extension context available. Suggest instead to:

1. Replace `context: Pick<vscode.ExtensionContext, 'globalState'>` with `context: Pick<vscode.ExtensionContext, 'globalState' | 'asAbsolutePath'>` in the signature of `createNewSamApp`.
2. Add the following to `samInitWizard.ts`:

```typescript
type CreateNewSamAppWizardArguments = Pick<vscode.ExtensionContext, 'asAbsolutePath'>
...
public constructor(
    private readonly args: CreateNewSamAppWizardArguments,
    ...
) {
```

3. Pass `args.asAbsolutePath` to `promptUserForRuntime`.

Reply:
Do we think that this or other custom buttons might be used outside the SAM app? I positioned the `buttons.ts` file in a way that it can be used in a more generic fashion, and so that way we can add additional buttons with different icons.

Are there any ways we could make this happen with relative paths instead of the extension absolute path?